recipes/TXM_Sandbox/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "TXM_Sandbox" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/fa/57/4d28f168e88f19781c615a8b7a0566dffb1365550a3570c7289d8bbc0a07/TXM_Sandbox-0.1.tar.gz 
```

Please use this url instead:

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/TXM_Sandbox-{{ version }}.tar.gz
```

---

conda cannot find the source file with the suggested url pattern. The error message is "RuntimeError: Could not download https://pypi.io/packages/source/T/TXM_Sandbox/TXM_Sandbox-0.1.1.tar.gz"

---

tried the suggested url format again and all tests are successfully passed.